nats-server/clientsz.yml

```diff
@@ -18,3 +18,24 @@ observation:
         source: limit
         type: long_gauge
         label: max channels
+
+  - name: clientsz_client_subscription
+    metricNamespace: nats
+    path: $.clients[?(@.id=${clientName}})].subscriptions.${subscriptionName}
+
+    metric:
+      - name: streaming.clients.subscriptions.pending.count
+        source: counter
+        type: long_gauge
+        label: pending count
+
+      - name: streaming.clients.subscriptions.sent
+        source: counter
```

Comment:
Same as above, mixup between source and type?

Reply:
Fixed